packages/payload/src/admin/components/forms/FieldDescription/int.spec.tsx

```diff
@@ -0,0 +1,79 @@
+import '@testing-library/jest-dom';
```

Comment:
@tyteen4a03 I wonder if we should be using an e2e test for this? Normally we don't use int tests to test the dom directly. Thoughts?

Reply:
My experience is with int tests - don't really know how e2e tests in Payload works.

Reply:
I exchanged the int test for an e2e. I also ran a linter so it adjusted a bunch of the files you touched, just fyi. Lastly I corrected an the type for the functional description as it was and always has been incorrect.